xml/security_ldap.xml

```diff
@@ -398,15 +415,13 @@ objectclass (2.16.840.1.113730.3.2.333 NAME 'nsPerson' <co xml:id="co-ldap-schem
      </listitem>
     </varlistentry>
   </variablelist>
-
-  <sect2 xml:id="sec-security-ldap-server-instance">
+ 
+    <sect2 xml:id="sec-security-ldap-server-instance">
   <title>Creating a &ds389; Instance with a Custom Configuration File</title>
     <para>
-     Use the <command>dscreate</command> command to create a new &ds389; instance.
-     There are several ways to create a new instance: interactively, from a 
-     template file, and using a custom configuration file.
-     In this section you will learn how to use a custom configuration file. This 
-     file must be in the INF format. You may name it anything you like.
+     You may create a new &ds389; instance from a simple custom configuration
+     file. This file must be in the INF format, and you may name it anything you
```

Comment:
👻.🍿.inf

Reply:
:beers:

Reply:
INF is correct, it is the name of the file format. The .inf extension is not required.

fixed dscreate/dsctl error https://github.com/SUSE/doc-sle/pull/732

Reply:
inf.✨